src/components/DownloadLoginFile.js

```diff
@@ -131,31 +137,14 @@ class DownloadLoginFile extends Nimiq.Observable {
     _onMouseDown(event) {
         /** @type {HTMLElement} */
         const target = (event.target);
-        // A click on the Continue button is already covered by a 'click' handler.
-        if (target.matches('.continue')) return;
+        // Clicks on the continue or download buttons are already covered by a 'click' handler.
+        if (target.matches('.continue') || target.matches('.download-button')) return;
```

Comment:
Still needed now that the mousedown handler is only bound to the login file?

Reply:
Probably not.